src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsEvolveMercury.java

```diff
@@ -18,7 +17,8 @@
   @Test
   public void adsEvolveMercury() {
     AdsEvolveObject wikiPage = new AdsEvolveObject(driver);
-    String testedPage = urlBuilder.getUrlForPage(EVOLVE_PAGE);
+//    String testedPage = UrlBuilder.createUrlBuilderForPage(EVOLVE_PAGE).getUrlForPage();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)